api/genomics/wdl/ReblockGVCF.wdl

```diff
@@ -1,31 +1,33 @@
 version 1.0
 
-# Sourced from https://github.com/broadinstitute/warp/blob/JointGenotyping_v1.4.0/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl
+# Sourced from https://github.com/broadinstitute/warp/blob/JointGenotyping_v1.4.0/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl but there's been significant divergence since then to support an array of inputs
 
 workflow ReblockGVCF {
 
   String pipeline_version = "1.1.0"
 
   input {
-    File gvcf
-    File gvcf_index
+    File gvcfs_list
+    String output_gcs_bucket
     String docker_image = "us.gcr.io/broad-gatk/gatk:4.1.8.0"
   }
 
-  String gvcf_basename = basename(gvcf, ".g.vcf.gz")
+  String output_gcs_bucket_no_trailing_slash = sub(output_gcs_bucket, "/$", "")
 
-  call Reblock {
-    input:
-      gvcf = gvcf,
-      gvcf_index = gvcf_index,
-      output_vcf_filename = gvcf_basename + ".reblocked.g.vcf.gz",
-      docker_image = docker_image
+  scatter (gvcf in read_lines(gvcfs_list)) {
```

Comment:
Out of curiosity, is this throttled in some manner by Cromwell? Or will it just attempt to queue up these tasks on the underlying execution system?

Reply:
I think it's the latter. From personal experience, cromwell seems to just spin them up ASAP.